src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/config/OrchestrationExceptionHandler.kt

```diff
@@ -27,6 +29,30 @@ class OrchestrationExceptionHandler {
       )
   }
 
+  @ExceptionHandler(BookerAuthFailureException::class)
+  fun handleBookerAuthFailureException(e: BookerAuthFailureException): ResponseEntity<ErrorResponse?>? {
```

Comment:
This should not be here we should not have merged this in.

Reply:
Yes, agreed. But it is not being used. Any subsequent changes I will merge to a separate branch for MVP.